pvlib/pvsystem.py

```diff
@@ -432,20 +452,54 @@ def sapm(self, effective_irradiance, temp_cell, **kwargs):
         """
         return sapm(effective_irradiance, temp_cell, self.module_parameters)
 
-    def sapm_celltemp(self, irrad, wind, temp):
-        """Uses :py:func:`sapm_celltemp` to calculate module and cell
-        temperatures based on ``self.racking_model`` and
-        the input parameters.
+    def sapm_celltemp(self, poa_global, temp_air, wind_speed,
+                      parameter_set=None):
+        """Uses :py:func:`temperature.sapm_cell` to calculate cell
+        temperatures.
 
         Parameters
         ----------
-        See pvsystem.sapm_celltemp for details
+        poa_global : numeric or Series
```

Comment:
`numeric or Series` seems to have crept back in or these were missed before. Should just be `numeric`. Applies to return statement and pvsyst method as well.

Reply:
Not sure how this happened, I must have not synched local files among my different computers, which worries me. Fixed now.